### PR TITLE
rename to pynessie

### PR DESCRIPTION
**Pull request by rymurr**
This change is [Reviewable](https://reviewable.io/reviews/projectnessie/nessie/234)

### CODE REVIEW

File: python/CONTRIBUTING.rst [hunk 1]
```diff
@@ -45,7 +45,7 @@ articles, and such.
 Submit Feedback
 ~~~~~~~~~~~~~~~
 
-The best way to send feedback is to file an issue at https://github.com/rymurr/nessie_client/issues.
+The best way to send feedback is to file an issue at https://github.com/rymurr/pynessie/issues.
```
Comment (laurentgo) [inline python/CONTRIBUTING.rst:48]: Wrong repository url?

File: python/CONTRIBUTING.rst [hunk 2]
```diff
@@ -57,17 +57,17 @@ If you are proposing a feature:
...
 
-Ready to contribute? Here's how to set up `nessie_client` for local development.
+Ready to contribute? Here's how to set up `pynessie` for local development.
 
-1. Fork the `nessie_client` repo on GitHub.
+1. Fork the `pynessie` repo on GitHub.
 2. Clone your fork locally::
 
-    $ git clone git@github.com:your_name_here/nessie_client.git
+    $ git clone git@github.com:your_name_here/pynessie.git
```
Comment (laurentgo) [inline python/CONTRIBUTING.rst:65]: same?

File: python/CONTRIBUTING.rst [hunk 3]
```diff
@@ -57,17 +57,17 @@ If you are proposing a feature:
 Get Started!
 ------------
 
-Ready to contribute? Here's how to set up `nessie_client` for local development.
+Ready to contribute? Here's how to set up `pynessie` for local development.
 
-1. Fork the `nessie_client` repo on GitHub.
+1. Fork the `pynessie` repo on GitHub.
```
Comment (laurentgo) [inline python/CONTRIBUTING.rst:62]: `nessie` repo

File: python/README.rst [hunk 4]
```diff
@@ -2,58 +2,6 @@
...
-* TODO
-
-Credits
--------
-
-This package was created with Cookiecutter_ and the `audreyr/cookiecutter-pypackage`_ project template.
-
-.. _Cookiecutter: https://github.com/audreyr/cookiecutter
-.. _`audreyr/cookiecutter-pypackage`: https://github.com/audreyr/cookiecutter-pypackage
+More information can be found at [projectnessie.org](http://projectnessie.org/).
```
Comment (laurentgo) [inline python/README.rst:7]: https://projectnessie.org

File: python/docs/installation.rst [hunk 5]
```diff
@@ -32,13 +32,13 @@ You can either clone the public repository:
 
 .. code-block:: console
 
-    $ git clone git://github.com/rymurr/nessie_client
+    $ git clone git://github.com/rymurr/pynessie
```
Comment (laurentgo) [inline python/docs/installation.rst:35]: wrong repository :)

**Review by laurentgo — CHANGES_REQUESTED** [review]

File: python/CONTRIBUTING.rst (hunk 1 above)
Comment (rymurr) [reply python/CONTRIBUTING.rst:48]: done

File: python/CONTRIBUTING.rst (hunk 3 above)
Comment (rymurr) [reply python/CONTRIBUTING.rst:62]: done

File: python/CONTRIBUTING.rst (hunk 2 above)
Comment (rymurr) [reply python/CONTRIBUTING.rst:65]: done

File: python/README.rst (hunk 4 above)
Comment (rymurr) [reply python/README.rst:7]: done

File: python/docs/installation.rst (hunk 5 above)
Comment (rymurr) [reply python/docs/installation.rst:35]: lol done

**Review by laurentgo — APPROVED** [review]
> Reviewed 4 of 36 files at r1.
> *[Reviewable](https://reviewable.io/reviews/projectnessie/nessie/234)* status: 4 of 38 files reviewed, 3 unresolved discussions (waiting on @jacques-n and @laurentgo)

**Pull request merged**